maap/maap.py

```diff
@@ -256,13 +256,76 @@ def cancelJob(self, jobid):
         job.id = jobid
         return job.cancel_job()
 
-    def listJobs(self, username=None, page_size=None, offset=None):
-        if username==None and self.profile is not None and 'username' in self.profile.account_info().keys():
+    def listJobs(self, username=None, 
+                       algo_id=None, 
+                       end_time=None, 
+                       get_job_details=True, 
+                       offset=None, 
+                       page_size=None, 
+                       priority=None, 
+                       queue=None,
+                       start_time=None,
+                       status=None,
+                       tag=None, 
+                       version=None):
+        """
+        Returns a list of jobs for a given user that matches query params provided.
+
+        Args:
+            username (str): Platform user.
+            algo_id (str): Algorithm type.
+            end_time (str, optional): Specifying this parameter will return all jobs that have completed from the provided end time to now. e.g. 2024-01-01 or 2024-01-01T00:00:00.000000Z.
+            get_job_details (bool, optional): Flag that determines whether to return a detailed job list or a compact list containing just the job ids and their associated job tags. Default is True.
+            offset (int, optional): Offset for pagination.
+            page_size (int, optional): Page size for pagination.
+            priority (int, optional): Job processing priority. Valid values are integers from 0-9.
```

Comment:
Currently, we do not use priorities on MAAP, would providing a value (0-9) accidentally prevent users from seeing their jobs ? If yes, then we should not expose this parameter to maap-py.

Reply:
Removed `priority` param.